<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RPG Character Creator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>575</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="mainMenu">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>271</width>
        <height>451</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-20</y>
         <width>241</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QPushButton" name="pbLoadChar">
          <property name="text">
           <string>Load a character</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pbNewChar">
          <property name="text">
           <string>Create a new character</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="sbCharNr">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pbExit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>420</y>
         <width>200</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="charList">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>300</width>
        <height>450</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="charCreate">
     <widget class="QGroupBox" name="gbRace">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>150</y>
        <width>200</width>
        <height>120</height>
       </rect>
      </property>
      <property name="title">
       <string>Race</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>114</width>
         <height>94</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rbHuman">
          <property name="text">
           <string>Human</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbElf">
          <property name="text">
           <string>Elf</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbOrc">
          <property name="text">
           <string>Orc</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gbClass">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>300</y>
        <width>200</width>
        <height>120</height>
       </rect>
      </property>
      <property name="title">
       <string>Class</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>114</width>
         <height>94</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="rbMage">
          <property name="text">
           <string>Mage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbArcher">
          <property name="text">
           <string>Archer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbWorrior">
          <property name="text">
           <string>Worrior</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>446</x>
        <y>80</y>
        <width>251</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leName"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pbMainMenu_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>470</y>
        <width>201</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Return to the main menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbCreateChar">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>201</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="charMenu">
     <widget class="QWidget" name="verticalLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>-10</y>
        <width>203</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QPushButton" name="pbSave">
         <property name="text">
          <string>Save to file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbItemMenu">
         <property name="text">
          <string>Show equipment menu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRemoveChar">
         <property name="text">
          <string>Remove character</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="charStats">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>300</width>
        <height>450</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pbMainMenu">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>470</y>
        <width>201</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Return to the main menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="itemMenu">
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>-10</y>
        <width>201</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="pbEquip">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Equip / unequip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRemoveItem">
         <property name="text">
          <string>Remove current item</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbNewItem">
         <property name="text">
          <string>Create a new item</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="itemStats">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>300</width>
        <height>450</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pbCharMenu">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>470</y>
        <width>201</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Return to the character menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbNextItem">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>520</y>
        <width>219</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Go to the next item</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="createItem">
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>60</y>
        <width>251</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leItemName">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="gbItemType">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>120</y>
        <width>291</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Type</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>291</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Weapon</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Armor</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pbCreateItem">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>201</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbCharMenu2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>470</y>
        <width>201</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Return to Item Menu</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stackedWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>statusbar</receiver>
   <slot>clearMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>587</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
